### PR TITLE
[EDIT] Key Value ADR 8

### PR DESCRIPTION
**Pull request by scottf**
Fine tuning KV Management section.

### CODE REVIEW

File: adr/ADR-8.md [hunk 1]
```diff
@@ -245,14 +247,18 @@ type KeyValueManager interface {
...
+    KeyValueBucketNames(ctx context.Context) KeyValueNamesLister
 
     // KeyValueBuckets is used to retrieve a list of key value bucket
     // statuses.
-    KeyValueBuckets(ctx context.Context) KeyValueLister
+    KeyValueBuckets(ctx context.Context) KeyValueStatusLister
 }
 ```
 
+`KeyValueBucketNames` provides a `KeyValueNamesLister`. `KeyValueBuckets` provides a `KeyValueStatusLister`.
```
Comment (Jarema) [inline adr/ADR-8.md:258]: I would mention here, that the KeyValue listers should have the same pattern used for returning the itemrs as the Stream listers, for consistency.

Comment (scottf) [reply adr/ADR-8.md]: I had this information in a separate paragraph below this block, so moved it inline.

**Review by Jarema — CHANGES_REQUESTED** [review]
Looks good, but some indentation issue popped up. Take a look:
https://github.com/nats-io/nats-architecture-and-design/blob/edit-kv-adr-8/adr/ADR-8.md

**Comment by scottf** [comment]
Fixed indentation

**Review by Jarema — APPROVED** [review]
LGTM!

**Pull request merged**